content/en/docs/configuration/external.md

```diff
@@ -5,58 +5,59 @@ weight: 50
 type: "docs"
 ---
 
-cert-manager supports external `Issuer` types. These external `Issuer` types are
-issuers that are not support by cert-manager by default, or are 'out of tree',
-however are treated the exact same as any other internal `Issuer` type. External
-issuer types are typically installed by deploying another pod into your cluster
-that will watch `CertificateRequest` resources and honor them based on
-configured `Issuer` resources. These issuer types exist outside of the
+cert-manager supports external `Issuer` types. While external issuers that not
+implemented in the main cert-manager repository, they are otherwise treated the
```

Comment:
```suggestion
cert-manager supports external `Issuer` types. While external issuers are not
implemented in the main cert-manager repository, they are otherwise treated the
```

Reply:
whoops, thanks. Fixed :+1: